src/Components/Patient/FileUpload.tsx

```diff
@@ -603,6 +606,11 @@ export const FileUpload = (props: FileUploadProps) => {
       });
   };
 
+  // upload file on click of custom button
+  const uploadFileFromCustomButton = () => {
+    document.getElementById("filePicker")?.click();
+  };
```

Comment:
I disagree with using `document.getElementById` to find the file picker and simulate a click on it. 

How about we use `ref` here? It's generally the [better](https://www.javascriptstuff.com/use-refs-not-ids/) approach.

```
...
const filePicker = React.useRef(null);
...
<input
    ref={filePicker}
...
filePicker.current.click();
...
```

Reply:
done